pkgs/servers/nextcloud/packages/apps/recognize.nix

```diff
@@ -0,0 +1,103 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  nodejs,
+  python311,
+  util-linux,
+  ffmpeg,
+}:
+stdenv.mkDerivation rec {
+  pname = "nextcloud-app-recognise";
+  version = "8.1.1";
+
+  srcs = [
+    (fetchurl {
+      inherit version;
+      url = "https://github.com/nextcloud/recognize/releases/download/v${version}/recognize-${version}.tar.gz";
+      hash = "sha256-RIa2HzX2hfgS7KHGGBsxEdUCqDqXeAs3Xi288qA1gU4=";
+    })
+
+    (fetchurl {
+      inherit version;
+      url = "https://github.com/nextcloud/recognize/archive/refs/tags/v${version}.tar.gz";
+      hash = "sha256-op1Fdm40d8V6V+rfne0ECr60xRjBYrBPrCD3kaNeFfY=";
+    })
+    (fetchurl {
+      inherit version;
+      url = "https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.9.1.tar.gz";
+      hash = "sha256-f1ENJUbj214QsdEZRjaJAD1YeEKJKtPJW8pRz4KCAXM=";
+    })
+
+  ];
+
+  unpackPhase = ''
+    # Merge the app and the models from github
+    tar -xzpf "${builtins.elemAt srcs 0}" recognize;
+    tar -xzpf "${builtins.elemAt srcs 1}" recognize-${version}/models;
+    mv recognize-${version}/models recognize
```

Comment:
I think we can also unpack the models right into that directory, right?

Reply:
yes, done